app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -16,13 +16,20 @@
     MetricData,
     SignificanceData,
 )
+from experimenter.jetstream.tests import mock_valid_outcomes
 from experimenter.jetstream.tests.constants import TestConstants
+from experimenter.outcomes import Outcomes
 
 
+@mock_valid_outcomes
 @override_settings(FEATURE_ANALYSIS=False)
 class TestFetchJetstreamDataTask(TestCase):
     maxDiff = None
 
+    def setUp(self):
+        super().setUp()
+        Outcomes.clear_cache()
```

Comment:
Very nice.  Could probably also stick this in `setUpClass` since it only needs to load once for this whole suite instead of per test, but not a big deal.